.ci-config/base-config.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
```

Comment:
what is diference of base-config.xml and just config.xml

I think it should be same file.

we should have config.xml and config-patch.xml

Reply:
done, removed base config